packages/data/src/world/oot/overworld.yml

```diff
@@ -2359,7 +2375,7 @@
   region: GERUDO_VALLEY
   time: flow
   exits:
-    "Gerudo Valley Falls": "true"
+    "Gerudo Valley Falls": "is_child || can_swim_or_sink"
```

Comment:
Writing here since I can't in the actual region, but Gerudo Valley After Bridge connecting to Gerudo Valley Falls needs to be changed to can_swim_or_sink.